### DISCUSSION TITLE
How to use Kawaii Physics in the Sequencer?

### DISCUSSION THREAD

**Discussion by Zizo908** [pafuhana1213/KawaiiPhysics#141] [Q&A] [answered]
Hi Artists Community,

I faced a big problem with it that I was trying to solve for 1 full day now, I made kawaii physics for a character but when I drag the animation blueprint to the sequencer and added the animation sequence to it, it keeps playing non stop and it doesn't respect the play or stop button in the sequencer. Even when I clicked render the character didn't move at all. (I didn't setup anything in the event graph , only I did kawaii physics in the AnimGraph of the blueprint).

How to make kawaii physics character work in sequencer? and What is the true way to import Kawaii physics inside the sequencer to render a scene or cinematic?

**Comment by Zizo908** [comment]
@pafuhana1213 Any thoughts about how can I achieve that?

**Comment by pafuhana1213** [comment, marked as answer]
Hi @Zizo908 ,
Sorry for my late reply. The KawaiiPhysics sample includes one that uses a Sequencer, so maybe that will help you. Another option is PostProcessAnimBP. Give it a try.

thanks!

**Comment by AirYEXXUAN** [comment]
我是对模拟好的动画将其烘焙成动画序列后再在序列器里进行播放的！
